README.md

```diff
@@ -1,150 +1,10 @@
 # Digital Humanities Final
 
-![Title Image](assets/images/intro-image.png)
+![Home Image](assets/images/Wager's_Action_off_Cartagena,_28_May_1708.jpg)
 
-## Instructor and Author
+The transatlantic slave trade, also known as triangle trade, was a product of the colonization and expansion of imperialistic European nations searching for new economic opportunities in the Americas. In most cases, goods would be shipped from Europe to Africa, slaves from Africa to the Americas, and goods from the Americas back to Europe, thus completing the triangle of trade. However, long slave voyages sometimes encountered problems associated with piracy. 
 
-[David J. Thomas](mailto:dave.a.base@gmail.com), [thePortus.net](http://thePortus.net)<br />
-Instructor of Ancient History and Digital Humanities<br />
-Department of History<br />
-[University of South Florida](https://github.com/usf-portal)
+Piracy have a long history dating back more then 2000 years, according to historian Alfred Bradford (1). Ravaging the seas in search of vast fortunes, pirates sealed their place in history as ruthlessly violent savages. One of the most popular eras in pirate history, an era of fascination among historians and the general public partially because of its place in popular culture, is known as the Golden Age (c.1700-c.1730). 
 
----
+One could assume based on the laws related to piracy and enacted by nations, like England and Spain during the early 1700’s, that piracy had a significant impact of the transatlantic trade. The intent of this project is to gage the economic impact that pirates had on the slave trade between 1700 and the mid 1800s, as well as to analyze how significant piracy was on the slave trade as a whole. By using data visualization software, this project will attempt to answer these questions in an effort to better understand the role of piracy in world history. 
```

Comment:
This is good as is, but I wonder if you might write a more powerful introduction, now that you have everything together and can see how the whole project looks.

If you want to use this as a portfolio for the future, you might consider taking another quick look at it, especially this last paragraph. I wonder if you might actually state it more strongly by just outright saying that despite what one might think based on works about piracy, the evidence doesn't suggest a strong impact on the slave trade. That would probably make a reader want to go further instead of the neutral thesis.